apps/kg/distributed/README.md

```diff
@@ -0,0 +1,26 @@
+## Training Scripts for distributed training
+
+1. Partition data
```

Comment:
We should tell users the partition.sh will generate a list of directories and files in the README.

Reply:
This readme will be re-write after moving into new repo and finish pip install.